src/actions/operator.ts

```diff
@@ -1086,18 +1088,19 @@ class ActionVisualReflowParagraph extends BaseOperator {
   public async run(vimState: VimState, start: Position, end: Position): Promise<VimState> {
     [start, end] = Position.sorted(start, end);
 
-    let textToReflow = TextEditor.getText(new vscode.Range(start, end));
-    let indent = this.getIndentation(textToReflow);
+    start = start.getLineBegin();
+    end = end.getLineEnd();
 
-    textToReflow = this.reflowParagraph(textToReflow, indent);
+    let textToReflow = TextEditor.getText(new vscode.Range(start, end));
+    textToReflow = this.reflowParagraph(textToReflow);
 
     vimState.recordedState.transformations.push({
       type: 'replaceText',
       text: textToReflow,
       start: start,
       end: end,
-      // Move cursor to front of line to realign the view
-      diff: PositionDiff.newBOLDiff(),
+      // Move cursor to front of lowest line to realign the view
+      diff: PositionDiff.newBOLDiff(textToReflow.split('\n').length - 1),
```

Comment:
This doesn't work when the cursor is already at the bottom (`gq5k`, for instance)

Reply:
@J-Fields Ah, that's really a problem. :-/ I'll revert it.